pkg/addon/manifests/chart/templates/service.yaml

```diff
@@ -1,4 +1,5 @@
 ---
+{{- if not (eq .Values.product "") }}
```

Comment:
not quite sure why this is needed, we do not create service when product is empty?

Reply:
the previous logic is that we create ClusterIP service firstly and then update the service when the claim is appended into the managedCluster by work-manager addon. 
for OCP 311, work-agent will meet error to update the service` invalid: spec.ports[0].nodePort: Forbidden: may not be used when type is 'ClusterIP'` because this issue https://github.com/kubernetes/kubectl/issues/221 has not fixed in old k8s.
current fix is that to create service until the product claim is appended to the managedCluster. will not need to update the service.

Reply:
it means we will not create service in *ks clusters since we cannot know the product? is this desired?

Reply:
no, the product claim will be Others for *ks not empty.